docs/3-v6-beta/6-cpi-example.md

```diff
@@ -1,9 +1,69 @@
 ---
 sidebar_label: CPI Example
-description: Coming Soon
-draft: true
+description: Integrate Jupiter Swap via CPI
 
 ---
 
-# CPI Example
-Coming Soon
\ No newline at end of file
+# Jupiter Swap via CPI
+
+Integrate your program with Jupiter Swap.
+
+CPI is one of two approaches to swap via Jupiter.
+
+:::info CPI Limitations
+As of August 2023, taking the CPI approach has some tradeoffs. Due Solana's transaction limit of 1232 bytes, swaps via CPI will likely fail at runtime since Jupiter routes may involve multiple dexes in order to reduce price impact. You could set a limit to the number of accounts used for swaps via Jupiter's swap API to fit it within your needs. However, limiting the accounts will likely incur greater price impact.
+:::
+
+:::info Use Flash-Fill
+Instead, we recommend taking the [flash-fill](/docs/v6-beta/flash-fill) approach. The flash-fill approach takes advantage of Versioned Transaction and Address Lookup Tables to allow for more accounts per transaction while keeping within the 1232 bytes limit.
+:::
+
+## Quick Example
+To ease integration via CPI, you may add the following crate [jupiter-cpi](https://github.com/jup-ag/jupiter-cpi) to your program.
+
+`Cargo.toml`
+```toml
+[dependencies]
+jupiter-cpi = { git = "https://github.com/jup-ag/jupiter-cpi", rev = "5eb8977" }
+... other dependencies
+```
+
+```rust
+use jupiter_cpi;
+
+...
+
+let signer_seeds: &[&[&[u8]]] = &[...];
+
+let accounts = jupiter_cpi::cpi::accounts::SharedAccountsRoute {
```

Comment:
should we just use the remaining account here? seem like quite hard to construct all accounts here

Reply:
yup. easier to get from the remaining accounts.

Reply:
Wanted to make the example explicit this way since there's no example client side code that sends the tx with remaining accounts.
 
What do you guys think of leaving it this way but adding a comment there that they can use remaining accounts?

Reply:
Something like this https://github.com/jup-ag/space-station/commit/407758d77e97f90fa6a6c41023343a0536f5c421